lib/widgets/screens/explore/recommended_mentors_filters.dart

```diff
@@ -106,9 +115,15 @@ class _RecommendedMentorsFilters extends State<RecommendedMentorsFilters>
 }
 
 class _Expertise extends StatefulWidget {
-  final Set<String> skills;
+  final List<String> skills;
```

Comment:
It looks like the parameter "skill" is what the backend and graphql models refer to as "textId". I'd consider renaming this for clarity, e.g. to expertiseTextId. (it's likely I've just become used to that naming convention, though, since it's what we use in the backend.)

Reply:
Good point. Renamed.